src/alda/repl/commands/ReplQuit.java

```diff
@@ -13,8 +14,14 @@
   @Override
   public void act(String args, StringBuffer history, AldaServer server,
                   ConsoleReader reader, Consumer<AldaScore> newInstrument) {
-    // Bye! =)
-    System.exit(0);
+    if (history.length() == 0 ||
+        Util.promptWithChoices(
+          reader,
+          "Are you sure you want to quit (destroying unsaved changes)?",
+          "yes", "no").equals("yes")) {
```

Comment:
Can we change the wording here to "You have unsaved changes. Are you sure you want to quit?"